magnum_cluster_api/cmd/image_loader.py

```diff
@@ -206,6 +209,7 @@ def _get_cloud_provider_images():
         "registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.28.0",
```

Comment:
Could you please add the 1.28.2 images in here?

Reply:
Done - added full set of images